3-transport/lessons/1-location-tracking/translations/assignment.bn.md

```diff
@@ -0,0 +1,17 @@
+# অন্যান্য জিপিএস ডেটা পর্যালোচনা করা
+
+## নির্দেশাবলী
+
+আমাদের জিপিএস সেন্সর থেকে আসা NEMA বাক্যগুলিতে লোকেশন ছাড়াও অন্যান্য ডেটা রয়েছে। অতিরিক্ত ডেটা অনুসন্ধান করে  আইওটি ডিভাইসে তা ব্যবহার করি।
+
+উদাহরণস্বরূপ - আমরা কী বর্তমান তারিখ এবং সময় পেতে পারি? আমরা যদি কোন মাইক্রোকন্ট্রোলার ব্যবহার করি, তবে  যেভাবে পূর্বের প্রজেক্টে এনটিপি সিগন্যাল ব্যবহার করেছি সেটাকে ব্যবহার করে জিপিএস ডেটা দ্বারা ঘড়ি সেট করা যাবে? আমরা কি উচ্চতা (সমুদ্রতল থেকে আমাদের উচ্চতা) বা বর্তমান গতি পেতে পারি?
+
+If you are using a virtual IoT device, then you can get some of this data by sending NMEA sentences generated using tools [nmeagen.org](https://www.nmeagen.org).
```

Comment:
Does this need translating?

Reply:
I translated this in the next line but forgot to remove the English sentence.   Thanks a lot for this catch. ❤️